webapp/src/features/Dashboard/SubscriptionTable.js

```diff
@@ -13,8 +13,11 @@ import Table from '../../widgets/Table'
 import './Dashboard.css'
 import Icons from '../../assets/icons'
 import classNames from 'classnames'
+import { useHistory } from 'react-router-dom'
+import SortingCustomIcon from '../../widgets/SortingCustomIcon'
 
 const SubscriptionTable = () => {
+	const history = useHistory()
 	const viewButton = (cell, row) => (
 		<div className='d-inline-flex align-items-center'>
 			{row.userName === 'Test1' ? (
```

Comment:
add the TODOs where we need to remove the mock data

Reply:
updated